llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -79,7 +79,6 @@ bool VPRecipeBase::mayWriteToMemory() const {
     return !cast<VPWidenCallRecipe>(this)
                 ->getCalledScalarFunction()
                 ->onlyReadsMemory();
-  case VPWidenIntrinsicSC:
```

Comment:
Why remove this?

Reply:
Dropped by accident, should be restored in a4819bd46d8baebc3aaa8b38f78065de33593199, thanks!